tools/publish.sh

```diff
@@ -19,10 +19,13 @@ pubDeb () {
         # read distro code from filename
         CODENAME=$(echo $PKG | grep -oP '[^-]+.deb$' | cut -d. -f1)
         jfrog bt upload --publish --override --deb "${CODENAME}/main/amd64" "$PKG" "$VERSION_PATH"
-        # we only build for 1st yearly releases. this section pushes the builds for next release version
-        NEXT_CODENAME=$(echo $DISTS | jq -rcM ".[index(\"${CODENAME}\")+1]")
-        if [ $NEXT_CODENAME != null ]; then
-            jfrog bt upload --publish --deb "${NEXT_CODENAME}/main/amd64" "$PKG" "$VERSION_PATH"
+        if [ $REPO == ubuntu ]; then
+            # we only build for 1st yearly releases. this section pushes the builds for next release version
+            NEXT_CODENAME=$(echo $DISTS | jq -rcM ".[index(\"${CODENAME}\")+1]")
+            if [ $NEXT_CODENAME != null ]; then
+                rename "${CODENAME}.deb" "${NEXT_CODENAME}.deb" "$PKG"
+                jfrog bt upload --publish --override --deb "${NEXT_CODENAME}/main/amd64" *${NEXT_CODENAME}.deb "$VERSION_PATH"
```

Comment:
i think this may be optimized somehow to use the same file for both $CODENAME and $NEXT_CODENAME dists without uploading duplicates. iirc, the web ui of bintray can do this.

Reply:
resolved by https://github.com/OpenBangla/OpenBangla-Keyboard/pull/130/commits/09b19f173dc1e8114c078127f59db8a6eb3a97fd. duplicates are now symlinked.